apps/web/playwright.config.ts

```diff
@@ -0,0 +1,87 @@
+import { defineConfig, devices } from '@playwright/test';
+import dotenv from 'dotenv';
+import path from 'path';
+
+dotenv.config({ path: path.resolve(__dirname, '.env.playwirght.test') });
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './tests',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: process.env.CI ? 'blob' : 'html',
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: 'http://127.0.0.1:4200',
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: 'on-first-retry',
+  },
+  timeout: 60_000,
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: 'chromium',
+      use: { ...devices['Desktop Chrome'] },
+    },
+    /*
+     * {
+     *   name: 'firefox',
+     *   use: { ...devices['Desktop Firefox'] },
+     * },
+     * {
+     *   name: 'webkit',
+     *   use: { ...devices['Desktop Safari'] },
+     * },
+     */
+
+    /* Test against mobile viewports. */
+    /*
+     * {
+     *   name: 'Mobile Chrome',
+     *   use: { ...devices['Pixel 5'] },
+     * },
+     * {
+     *   name: 'Mobile Safari',
+     *   use: { ...devices['iPhone 12'] },
+     * },
+     */
+
+    /* Test against branded browsers. */
+    /*
+     * {
+     *   name: 'Microsoft Edge',
+     *   use: { ...devices['Desktop Edge'], channel: 'msedge' },
+     * },
+     * {
+     *   name: 'Google Chrome',
+     *   use: { ...devices['Desktop Chrome'], channel: 'chrome' },
+     * },
+     */
+  ],
+
+  /* Run your local dev server before starting the tests */
+  /*
+   * webServer: {
+   *   command: 'npm run start',
+   *   url: 'http://127.0.0.1:3000',
+   *   reuseExistingServer: !process.env.CI,
+   * },
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
**note (non-blocking):** ‏Not for now, but maybe could be nice letting playwright setup the API test env here.

Reply:
haven't checked this option, but seems like we can run all the services and set the `url` option just for the `API`